admin/nload/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nload
+PKG_VERSION:=0.7.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.roland-riegel.de/nload/
+PKG_HASH:=c1c051e7155e26243d569be5d99c744d8620e65fa8a7e05efcf84d01d9d469e5
+
+PKG_MAINTAINER:=Jacky Guo <leickwell@Hotmail.com>
```

Comment:
`Hotmail`? I would assume `hotmail`?

Reply:
It's a bit difficult cause I type "Hotmail" when register GitHub.

Reply:
> `Hotmail`? I would assume `hotmail`?

OK, I'm fixed.